src/app/route/ProductPage/ProductPage.component.js

```diff
@@ -60,16 +73,34 @@ export default class ProductPage extends PureComponent {
                   parameters={ parameters }
                   areDetailsLoaded={ areDetailsLoaded }
                   configurableVariantIndex={ configurableVariantIndex }
+                  getRequiredCustomizableOptions={ getRequiredCustomizableOptions }
+                  getSelectedCustomizableOptions={ getSelectedCustomizableOptions }
+                  requiredCustomizableOptions={ requiredCustomizableOptions }
+                  customizable_options={ customizable_options }
+                  customizable_options_multi={ customizable_options_multi }
```

Comment:
Too many props, can we group them some-how? 5 more props is not okey. Try preparing them in reducer, understand which data-structure will be more suitable for your needs.